internal/daemon/path_manager.go

```diff
@@ -0,0 +1,33 @@
+package daemon
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/openshift/dpu-operator/dpu-cni/pkgs/cnitypes"
+)
+
+type PathManager struct {
```

Comment:
Why do we need this wrapper for Paths? When the paths can be made to consts.

Reply:
This allows changing all the path by passing in a differnt `rootDir`. The paths used to be consts, but that means we can't change them dynamically.